### PR TITLE
Fix issue with checking if path is root on windows

### PR DESCRIPTION
**Pull request by bertyhell**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -23,7 +24,8 @@ module.exports = function (startPath, ignore) {
 		} catch (err) {}
 
 		nextPath = path.join(searchPath + '/..');
-		if (nextPath === path.normalize('/') || nextPath === '.' || nextPath === '..') {
+		const root = (os.platform() === 'win32') ? nextPath.split(path.sep)[0] + path.sep : path.normalize('/');
```
Comment (maxrimue) [inline lib/index.js]: Thanks a lot for the pull request! Just one tiny nit: Could you perhaps add a little comment here that clarifies this line? Something like "if Windows, uses main drive's identifier as root path"? Thanks!

Comment (bertyhell) [reply lib/index.js]: > Thanks a lot for the pull request! Just one tiny nit: Could you perhaps add a little comment here that clarifies this line? Something like "if Windows, uses main drive's identifier as root path"? Thanks!

Sure, something like this?
https://github.com/bertyhell/parent-package-json/blob/a69451531fba2ed578e54336ba39c906296ab9e7/lib/index.js#L27-L28

Comment (maxrimue) [reply lib/index.js]: Yup, looks good to me, thanks! :+1:

**Comment by maxrimue** [comment]
Patch v2.0.1 is out with your fix, thanks again!

**Pull request merged**